docs/src/components/HeadingWithLink.tsx

```diff
@@ -1,24 +1,43 @@
 import React from "react";
-import styled from "styled-components";
-import { ButtonLink, Stack } from "@kiwicom/orbit-components";
+import styled, { css } from "styled-components";
+import { Stack } from "@kiwicom/orbit-components";
 import { Link as LinkIcon } from "@kiwicom/orbit-components/icons";
 import { SpaceAfter } from "@kiwicom/orbit-components/lib/common/common";
 
 import { getTextFromChildren, slugify } from "../utils/common";
 
 export const StyledAnchorWrapper = styled.div`
-  margin-top: -10px;
-
   svg {
     opacity: 0;
-    transition: opacity ${({ theme }) => theme.orbit.durationNormal} ease-in;
   }
 
   &:hover svg {
     opacity: 1;
   }
 `;
 
+const StyledAnchor = styled(({ className, children, title, href }) => (
+  <a className={className} title={title} href={href}>
+    {children}
+  </a>
+))`
+  ${({ theme }) => css`
+    display: inline-block;
+    padding: 10px;
+    margin: -10px;
+    margin-left: 1px;
+    :hover,
+    :active,
+    :focus {
+      outline: none;
+      svg {
+        transition: fill ${theme.orbit.durationFast} ease-in;
```

Comment:
I'd prefer transitioning both ways. Transitioning only on hover makes the fill change instantaneously when mouse is moved away.